### DISCUSSION TITLE
Error when fit_transform multilingual model. "HfHubHTTPError: 404 Client Error ... Sorry, we can't find the page you are looking for."

### DISCUSSION THREAD

**Discussion by danqroz** [MaartenGr/BERTopic#1246] [Q&A] [answered]
Hi everyone!
I used BERTopic many times in colab and today I started a new project and got the following error:
"HfHubHTTPError: 404 Client Error: Not Found for url: https://huggingface.co/api/models/sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2 (Request ID: Root=1-645a5904-05fa9d0a1620b9f1743d37fa)

Sorry, we can't find the page you are looking for."

Can someone help me? The error occours when run the `fit_transform()`

##### CODE
```
    !pip install bertopic pandas
    
    from google.colab import drive
    drive.mount('/content/drive')
    
    import pandas as pd
    from bertopic import BERTopic 
    
    DATA_PATH = "/content/drive/MyDrive/data/dasa/"
    
    df_user = pd.read_csv(DATA_PATH )

    docs = df_user["clean_text"].to_list()
    
    model = BERTopic(
        language="multilingual",
        calculate_probabilities=True,
        verbose=False,
        top_n_words=15,
        n_gram_range=(1, 2),
    )
    model.fit_transform(docs)
```

![error1](https://github.com/MaartenGr/BERTopic/assets/75531272/df4ac1e3-85d5-4071-9b2b-4707f74f1cb8)
![error2](https://github.com/MaartenGr/BERTopic/assets/75531272/fb385fa9-bf77-4536-9230-1b19ce992c74)

It seems that the standart model isn't available anymore so I tried different models but got the same error, like: 
```
    !pip install sentence-transformers[distiluse-base-multilingual-cased-v2]
    
    from sentence_transformers import SentenceTransformer
    
    model = BERTopic(embedding_model=SentenceTransformer('distiluse-base-multilingual-cased-v2'))
```

**Comment by MaartenGr** [comment, marked as answer]
This is a [known issue with HuggingFace](https://twitter.com/huggingface/status/1655760648926642178) at the moment. You can find more information about that [here](https://github.com/UKPLab/sentence-transformers/issues/1915). I would advise waiting until HuggingFace has restored those models. You could also see if you have any of the models cached and load them locally but I am quite sure HuggingFace is working hard on a fix seeing the large dependencies of OSS on sentence-transformers.

**Reply by danqroz** [reply]
> This is a [known issue with HuggingFace](https://twitter.com/huggingface/status/1655760648926642178) at the moment. You can find more information about that [here](https://github.com/UKPLab/sentence-transformers/issues/1915). I would advise waiting until HuggingFace has restored those models. You could also see if you have any of the models cached and load them locally but I am quite sure HuggingFace is working hard on a fix seeing the large dependencies of OSS on sentence-transformers.

Thank you!
